charts/hivemq-platform/values.yaml

```diff
@@ -88,17 +88,96 @@ restApi:
   # Enables or disables authentication and authorization.
   authEnabled: false
 
-# Configures the HiveMQ License information.
+# Configures the HiveMQ Platform license information.
 license:
   # Creates a default HiveMQ Platform license as a Kubernetes Secret.
   create: false
-  # The name of a secret with the license information.
+  # The name of a Kubernetes Secret with the HiveMQ Platform license information.
   # To create a new secret with a license, set create=true. Otherwise, false to reuse an existing one.
   name: ""
-  # Inlines the License information as an encoded 64-bytes string.
+  # Inlines the HiveMQ broker license information as an encoded 64-bytes string.
   data: ""
-  # Overrides the License information via file using --set-file license.lic.
+  # Overrides the HiveMQ broker license information via file using --set-file license.lic.
   overrideLicense: ""
+  # HiveMQ Enterprise extension licenses to include as part of the same Kubernetes Secret license.
+  extensions: {}
```

Comment:
A couple of important to notes regarding the implementation details:

1. Since the HiveMQ broker does not support checking extension licenses in subfolder (default license folder is always `/license`), we cannot mount multiple volumes so the same Secret used for the license broker needs to be used here (`subPath` is not an option here as change there won't be reflected on runtime for the broker to reload it).
2. The existing `extensions` values cannot be used to include a new sub-value called `license` or something like that. This is because the type of this `extensions` value is `array` and thus it makes impossible to use the existing feature for the license to inline a file license into the value, i.e. `--set-file extensions.hivemq-kafka-extension.license.elic`. For this to work, we would need to switch it from `array` to `object` as it is already defined under the new element `licenses.extensions` but unfortunately this would be a breaking change. So for that I created a new value `license.extensions` as `object` so this can be done.